dev_tools/nyaml2nxdl/nyaml2nxdl_forward_tools.py

```diff
@@ -253,13 +257,70 @@ def check_for_mapping_char_other(text):
     return text.strip()
 
 
-def xml_handle_doc(obj, value: str, line_number=None, line_loc=None):
+def Handle_each_part_doc(text):
```

Comment:
```suggestion
def parse_docstring_parts(text):
```

Should at least be following pythonic naming conventions (lower case).

Reply:
That was also unintended.